### PR TITLE
Options to output width/height instead of background-size in SCSS mixin

### PR DESCRIPTION
**Pull request by xposedbones**
In multiple scenario you need to enter the width/height of the svg instead of a background-size. I added an option to the mixin to output the width/height OR the background-size depending of the flag `$use-background-size`.

```scss
.hello {
    @include sprite('test', (), true, false);
    content: '';
}
```

Will output

```scss
.hello {
    content: '';
    background-image: (data:image...);
    width: 15px;
    height: 15px;
}
```

### CODE REVIEW

**Comment by xposedbones** [comment]
In order to not break existing code, I think having `@include-size` equals `width-height` would be the best options. I'll commit that tomorrow

**Comment by xposedbones** [comment]
@cascornelissen Pushed!  I think `$include-size: 'box'` makes the most sense here but I'm super open to use another keyword :)

**Comment by cascornelissen** [comment]
Nice, LGTM. Would you mind adding a test and updating the docs?

**Comment by xposedbones** [comment]
> Nice, LGTM. Would you mind adding a test and updating the docs?

Sorry for the delay, personal stuff came up. 

I can update the docs but I'm not sure how I'd write a test for this :/

**Comment by xposedbones** [comment]
@cascornelissen  I'm looking at the docs and would write it in `docs/variables.md` but I'm not sure it belongs there. Should we create a new documentation for the sass mixin?

I first learned about the third argument by looking directly at the mixin because I was curious how it worked under the hood, there was no documentation about the third argument.

**Comment by cascornelissen** [comment]
Interesting, looks like we could use some docs improvements at [the end of the document](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/blob/master/docs/options.md#stylesvariablesmixin--sprite) explaining what the Sass mixin is about I guess, I could also include that after merging this PR.

With regards to the tests, I'm guessing (hoping actually) that some tests are currently failing because they test equality with files in [this directory](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/tree/master/tests/output/styles). Updating these files and making sure all tests are green should be enough in this case.

**Comment by xposedbones** [comment]
Wow that fell through the cracks haha.  

I updated the tests and not a single test fails now :)

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@
 /*.scss
 /*.sass
 /*.less
+.DS_Store
```
Comment (cascornelissen) [inline .gitignore:11]: Could you remove this change from this PR? You should probably add this to your [global `.gitignore`](https://stackoverflow.com/a/7335487/1358948).

**Comment by cascornelissen** [comment]
No problem at all, if you could fix my last comment I'll make sure this gets merged and released as soon `3.6.x` (Webpack 5 support) is finished.

File: .gitignore (hunk 1 above)
Comment (xposedbones) [reply .gitignore:11]: Yeah of course I can remove it.

Comment (xposedbones) [reply .gitignore]: Done!

**Review by cascornelissen — APPROVED** [review]

**Comment by cascornelissen** [comment]
This feature was included in release [`3.7.0`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.7.0), thanks for your contribution! 🚀

**Pull request merged**